hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/response/snapshot/OMSnapshotCreateResponse.java

```diff
@@ -68,5 +73,16 @@ public void addToDBBatch(OMMetadataManager omMetadataManager,
     // Add to db
     omMetadataManager.getSnapshotInfoTable().putWithBatch(batchOperation,
         key, snapshotInfo);
+
+    // Remove all entries from renamedKeyTable
+    TableIterator<Long, ? extends Table.KeyValue<Long, RepeatedOmString>>
+        iterator = omMetadataManager.getRenamedKeyTable().iterator();
+
+    while (iterator.hasNext()) {
+      Long objectID = iterator.next().getKey();
+      omMetadataManager.getRenamedKeyTable()
+          .deleteWithBatch(batchOperation, objectID);
+    }
```

Comment:
This should be done in `createOmSnapshotCheckpoint` like in #4280 . For this PR you can put a TODO here since the other PR is not merged yet (which has the locking and deleteRange implementation).

And it should only remove entries in the snapshot scope in `renamedKeyTable`, similar to `deletedTable`.